### PR TITLE
First version of CI publishing on space

### PR DESCRIPTION
**Pull request by Avvessalom**

### CODE REVIEW

File: build.gradle.kts [hunk 1]
```diff
@@ -31,9 +31,11 @@ plugins {
 
 val kotlinVersion: String by project
 val targetJvm: String by project
+val publishingVersion: String by project
 
 subprojects {
-  group = "org.jetbrains"
+  group = "org.jetbrains.projector"
+  version = publishingVersion
```
Comment (SerVB) [inline build.gradle.kts:38]: In projector-server, we specify the version a bit differently to projector-client. Please see https://github.com/JetBrains/projector-server/pull/63 with the fix

File: gradle.properties [hunk 2]
```diff
@@ -26,9 +26,11 @@ intellijPluginVersion=1.1.2
 javassistVersion=3.27.0-GA
 kotlinVersion=1.5.20
 mockitoKotlinVersion=3.2.0
-projectorClientVersion=021a2cda
-projectorClientGroup=com.github.JetBrains.projector-client
+projectorClientVersion=fa179d0b
+projectorClientGroup=org.jetbrains.projector
```
Comment (SerVB) [inline gradle.properties:30]: I suggest changing the group only on CI. Please see https://github.com/JetBrains/projector-server/pull/63 with the fix

File: projector-server/build.gradle [hunk 3]
```diff
@@ -78,12 +84,12 @@ if (project.rootProject.file('local.properties').canRead()) {
 }
 
 // relay arguments
-def relayURL=localProperties["ORG_JETBRAINS_PROJECTOR_SERVER_RELAY_URL"]
-def serverId=localProperties["ORG_JETBRAINS_PROJECTOR_SERVER_RELAY_SERVER_ID"]
+def relayURL = localProperties["ORG_JETBRAINS_PROJECTOR_SERVER_RELAY_URL"]
+def serverId = localProperties["ORG_JETBRAINS_PROJECTOR_SERVER_RELAY_SERVER_ID"]
```
Comment (SerVB) [inline projector-server/build.gradle:88]: Please don't change formatting in those lines that you don't edit

**Review by SerVB — CHANGES_REQUESTED** [review]
Let me close this PR in favor of https://github.com/JetBrains/projector-server/pull/63

File: projector-server/build.gradle (hunk 3 above)
Comment (Avvessalom) [reply projector-server/build.gradle:88]: sorry, I just use `ctrl+alt+L`

Comment (Avvessalom) [reply projector-server/build.gradle:88]: sorry, I just use `ctrl+alt+L`

Comment (SerVB) [reply projector-server/build.gradle:88]: Well, for some reason we have different code style settings for the project. We should address this later, adding a linter on CI.

Anyway, the recommended commit actions are [the following](https://jetbrains.github.io/projector-client/mkdocs/latest/about/contributing/#submitting-changes) and they reformat only changed lines:

image

You can try use only it for now.

**Pull request closed without merging**